README.md

```diff
@@ -42,75 +52,124 @@ The repository is structured to support multi-team collaboration while maintaini
-  - `summary` A short summary of what the alert has detected and why.
-  - `__dashboardUid__` and `__panelId__`: Link to a dashboard and panel to facilitate alert investigation.
-    - The `__dashboardUid__` should be taken from the dashboard definition or copied from existing link in Grafana.
-    - `__panelId__` reference to the panel in the dashboard referenced by the `__dashboardUid__` annotation.
-  - `dashboardQueryParams` references extra query params to add to the dashboard url like the orgId or dashboard variables. `orgId=` is mandatory if the `__dashboardUid__` is set. If the dashboard is public (in the Shared Org), then use `orgId=1`, otherwise `orgId=2`.
-  - `dashboardExternalUrl`: dashboard external url links to an external grafana instance like grafana cloud.
+We follow standardized practices for organizing our alerts using PrometheusRule custom resources.
 
-* Mandatory labels:
-   - `area`
-   - `team`
-   - `severity`
-   - `cluster_id`
-   - `installation`
-   - `pipeline`
-   - `provider`
+#### Mandatory annotations
+- `description`: Detailed explanation of what happened and what the alert is monitoring
+- `runbook_url`: Link to a runbook page with incident management instructions
 
-* Optional labels:
-   - `cancel_if_.*`
+#### Recommended annotations
+- `summary`: Brief overview of what the alert detected
+- `__dashboardUid__`: Unique identifier of the relevant dashboard
+- `__panelId__`: Specific panel ID within the referenced dashboard
+- `dashboardQueryParams`: Additional URL parameters (must include `orgId=1` for Shared Org dashboards or `orgId=2` for others)
+- `dashboardExternalUrl`: Optional link to an external Grafana instance (like Grafana Cloud)
 
-#### Dashboard address
+##### Dashboard URL construction
 
-The dashboard url generated by alertmanager (for opsgenie and slack alerts) follows this rule:
+Alert manager generates dashboard URLs for Opsgenie and Slack alerts using these rules:
 
-1. if only `__dashboardUid__` is set, then the url is `https://grafana.domain/__dashboardUid__`
-2. if `__dashboardUid__` and dashboardQueryParams are set, then the url is `https://grafana.domain/__dashboardUid__?dashboardQueryParams`
-3. if `dashboardExternalUrl` is set then the url is `dashboardExternalUrl`
+1. With only `__dashboardUid__`: `https://grafana.domain/__dashboardUid__`
+2. With both `__dashboardUid__` and `dashboardQueryParams`: `https://grafana.domain/__dashboardUid__?dashboardQueryParams`
+3. If `dashboardExternalUrl` is set: Uses the exact URL provided
 
-#### Specific alert labels
+#### Mandatory labels
+- `area`: Functional area (e.g., platform, apps)
+- `team`: Responsible team identifier
+- `severity`: Alert severity level (page, notify)
+- `cluster_id`: Associated cluster ID
+- `installation`: Installation identifier
+- `pipeline`: Deployment pipeline
+- `provider`: Cloud provider
```

Comment:
Are these added automatically by the o11yplat?

Reply:
They are today yes (by the scraping agent) but we know provider is not always accurate

Reply:
I would but it;s not added by our logging pipeline :D

Reply:
I think I will just remove them actually